src/Utilities/Compiler/Options/MSBuildPropertyOptionNames.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Analyzer.Utilities
+{
+    /// <summary>
+    /// MSBuild property names that are required to be threaded as analyzer config options.
+    /// </summary>
+    internal static partial class MSBuildPropertyOptionNames
+    {
+        public const string TargetFramework = "TargetFramework";
```

Comment:
Another property we will need is `TargetPlatformMinVersion`

Reply:
Sure, I have pushed a commit for it. Feel free to add any required properties to this type, and the next package should ensure they are available to the analyzers.

For `TargetPlatformMinVersion`, I notice it is actually not set for default C# classlib project. Is that expected?

![image](https://user-images.githubusercontent.com/10605811/84957048-4153d580-b0af-11ea-9ea6-7a009af6813c.png)

Reply:
Yes it is optional, so i wouldn't expect it defaulted